src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -684,4 +684,8 @@ public void setRespondingInstructorList(Set<String> respondingInstructorList) {
     public void setRespondingStudentList(Set<String> respondingStudentList) {
         this.respondingStudentList = respondingStudentList;
     }
+
+    public String getEndTimeInIso8601Format() {
+        return TimeHelper.convertAndFormatDateToIso8601Utc(endTime, timeZone);
```

Comment:
Actually on second thoughts, I still feel that the conversion and the formatting should be separated (follow how you did `getStartTimeString` and `getEndTimeString`). The method that formats to ISO8601 shouldn't do the conversion.